pkgs/development/python-modules/python-dotenv-0_21/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, click
+, fetchPypi
+, ipython
+, mock
+, pytestCheckHook
+, pythonOlder
+, sh
+}:
+
+buildPythonPackage rec {
+  pname = "python-dotenv";
+  version = "0.21.0";
```

Comment:
I really dislike doing this but manim-voiceover is expecting a version < 0.22 and >= 0.21 and I don't know how to do that properly.
This file is taken from commit 000bf335745ae7af84e17ff6be401867e9180a5d of nixpkgs